common/lib/xmodule/xmodule/tests/test_graders.py

```diff
@@ -294,5 +281,25 @@ def test_grader_from_conf(self):
         self.assertAlmostEqual(graded['percent'], 0.11)
         self.assertEqual(len(graded['section_breakdown']), 12 + 1)
 
-        # TODO: How do we test failure cases? The parser only logs an error when
-        # it can't parse something. Maybe it should throw exceptions?
+    @ddt.data(
+        # empty
```

Comment:
Nit for debate: I think it would be cleaner to see the comment explaining what each ddt case is doing inside the bounding `()` for that case, instead of immediately preceding it.